hera_sim/noise.py

```diff
@@ -113,6 +113,14 @@ def resample_Tsky(fqs, lsts, Tsky_mdl=None, Tsky=180.0, mfreq=0.18, index=-2.5):
     """
     if Tsky_mdl is not None:
         tsky = Tsky_mdl(lsts, fqs)  # support an interpolation object
+
+        if tsky.shape != (len(lsts), len(fqs)):
```

Comment:
I think this is assuming that if tsky doesn't have dimensions of `(NTIMES, NFREQS)`, it has dimensions of `(NFREQS, NTIMES)` so transposing it will give sensible output--should we also worry about when tsky just has nonsensical dimensions (i.e., neither `(NTIMES, NFREQS)` nor `(NFREQS, NTIMES)`)?

Reply:
Good catch. Check the newest commit -- raises an error if it's a weird shape, but a warning if it can fix it.